src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/__snapshots__/HtmlToTextUtilsTests.HtmlToText_HorizontalLineElement.snap

```diff
@@ -0,0 +1,5 @@
+﻿Test paragraph 1
+
+-----
```

Comment:
I didn't know if the desired behaviour might be to stretch this to either the width of the preceding line or the width of the following line, or indeed perhaps the width of the widest line in the doc??  I'm sure if it's desired, we'll hear about it as this begins to be used :)

Reply:
Would probably say it'd be a little unsightly to have it stretch across the entire page, particularly as some text lines could just be reallllly long. If you view the text file in an editor with soft wrapping, this could also end up rendering something like:

```
----------------- <-- hits the soft wrap limit here
------            <-- overflows onto the next line
```

I'll just increase the length to like 15 characters :smile:

Reply:
Haha OK :)